src/coord/src/catalog/builtin.rs

```diff
@@ -128,7 +130,7 @@ impl Fingerprint for BuiltinFunc {
     }
 }
 
```

Comment:
We should probably add back in this comment below
https://github.com/MaterializeInc/materialize/blob/3c9ea8fb70154c2ef72addddcb864e38a0d74dc8/src/coord/src/catalog/builtin.rs#L135-L139

Reply:
Good call; will look at these docs